### DISCUSSION TITLE
How to diff specified commitID

### DISCUSSION THREAD

**Discussion by zuston** [extrawurst/gitui#1297] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

Firstly thanks for the great work on this project. I want to use this tool to diff the previous commitID with the latest commitID, but I didn't found the way. 

Like the command of `git diff prev-commitID`

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by extrawurst** [comment, marked as answer]
You can select/mark two commits in the log view and compare them
